src/Cli.Tests/VerifyExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Cli.Tests;
+
+internal static class VerifyExtensions
```

Comment:
Please add summary of the class and methods

Reply:
Class has been removed as we're now using the official Verify feature.